h/static/scripts/create-group-form.tsx

```diff
@@ -0,0 +1,63 @@
+import { render } from 'preact';
```

Comment:
For Preact components the convention is to name the modules after the name of the main component and put them in a `components` directory (with additional hierarchy as needed). Non-component modules use a kebab-case naming convention. Was this something you planned to do as part of making the UI actually functional?

Reply:
Done. I went with:

* `h/static/scripts/group-forms/index.tsx`
* * `h/static/scripts/group-forms/components/CreateGroupForm.tsx`
* `h/static/styles/group-forms.css`
* `assets.ini` bundle names: `group_forms_css` and `group_forms_js`